pkg/sinks/slack.go

```diff
@@ -43,8 +43,12 @@ func buildSlackMessage(kind, name, details, k8sgptCR string) SlackMessage {
 	}
 }
 
-func (s *SlackSink) Configure(config v1alpha1.K8sGPT, c Client) {
-	s.Endpoint = config.Spec.Sink.Endpoint
+func (s *SlackSink) Configure(config v1alpha1.K8sGPT, c Client, secret string) {
+	s.Endpoint = secret
```

Comment:
Wouldn't this set the endpoint to the secret name rather than a URL?

Reply:
No, what is passed to the `sink.Configure` is the [secret value](https://github.com/k8sgpt-ai/k8sgpt-operator/blob/d85401b2d6dc8806886c7bba36f173aaa7358f37/controllers/k8sgpt_controller.go#L307). I think I'll change the variable name. It's a bit misleading